helm/exporter-kube-state/templates/clusterrole.yaml

```diff
@@ -23,6 +23,8 @@ rules:
   - replicationcontrollers
   - limitranges
   - persistentvolumeclaims
+  - persistentvolumes
```

Comment:
Might be on purpose, but you are missing `- namespaces`

Reply:
`namespaces` already exists in this file.